pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -1580,13 +1581,25 @@ func createMarkAndAcceptChain(aclArgs types.AppNetworkACLArgs,
 		return errors.New("Invalid chain creation")
 	}
 
+	chainFlush := []string{"-t", "mangle", "--flush", name}
+
 	newChain := []string{"-t", "mangle", "-N", name}
 	log.Functionf("createMarkAndAcceptChain: Creating new chain (%s)", name)
 	err := iptables.IptableCmd(log, newChain...)
 	if err != nil {
-		log.Errorf("createMarkAndAcceptChain: New chain (%s) creation failed: %s",
+		// if chain already exists, we can skip this error
+		if !strings.Contains(err.Error(), "Chain already exists") {
+			log.Errorf("createMarkAndAcceptChain: New chain (%s) creation failed: %s",
+				name, err)
+			return err
+		}
+		log.Warningf("createMarkAndAcceptChain: New chain (%s) creation failed skipped: %s",
 			name, err)
-		return err
+		if err := iptables.IptableCmd(log, chainFlush...); err != nil {
+			log.Errorf("createMarkAndAcceptChain: Flush exists chain (%s) failed: %s",
+				name, err)
+			return err
+		}
```

Comment:
Don't you need to redo the newChain command here to make sure the chain is created?

Reply:
No, on this step we have chain (already exist) and flush rules of it (without delete of chain itself).

Reply:
ok